.github/workflows/e2e-tests.yml

```diff
@@ -2,6 +2,13 @@ name: e2e-tests
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths-ignore:
+      - 'docs/**'
+      - '.gitignore'
+      - 'README.md'
+      - '*.md'
+      - '*.txt'
```

Comment:
Wouldn't this also include https://github.com/google/timesketch/blob/master/requirements.txt ? But if we have changes of dependencies the e2e tests are very important.